src/main/java/uk/gov/companieshouse/web/accounts/controller/fullaccounts/FullAccountsCriteriaController.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.web.accounts.controller.fullaccounts;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+
+@Controller
+@RequestMapping("/accounts/full-accounts-criteria")
+public class FullAccountsCriteriaController extends BaseController{
+
+    @Value("${chs.url}")
+    private String chsUrl;
+
+    @GetMapping
+    public String getCriteria(Model model){
+        model.addAttribute("hideUserBar" , true);
+        return getTemplateName();
+    }
+
+    @PostMapping
+    public String postFullAccountsCriteria(){
+        return UrlBasedViewResolver.REDIRECT_URL_PREFIX + chsUrl;
```

Comment:
Does this journey redirect a user to the CHS search page?

Reply:
Yes it does following the prototype. In the future it will likely point to the Company Number page which has yet to be built.

Reply:
Fair enough!